static/src/inline-svgs/notifications-explainer-desktop.svg

```diff
@@ -0,0 +1 @@
atters after Senate vote</text><path fill="none" d="M263.8 226h11.4v1h-11.4z"/><text transform="translate(263.753 227.016)" fill="#BDBDBD" font-family="'ArialMT'" font-size="1.4">theguardian.com</text><path fill="#767676" d="M294.6 220.7l-.8-.7h.2l.7.6z"/><path fill="#767676" d="M294.8 220l-.7.7.6-.8z"/><path fill="#FFF" d="M255.7 221c-.2 0-.4.2-.5.4v1.4l.6.4-.2.2-.2.2-.2.3v.7l.6.3h-.4l-.3.2s0 .2-.2.3v.3c-.3-.2-.5-.5-.7-1v-1-1l.4-.8c0-.2.3-.4.5-.6 0-.2.4-.4.7-.5zm1.8 4.2c.2 0 .3 0 .5.2v.7c-.3.3-.6.3-1 .3s-.7 0-1-.2v-.3-.3l.4-.2h1zm2.5-1.5c0 .5 0 1-.3 1.2 0 .3-.3.6-.6 1 .2-.3.2-.5.2-.8 0-.4 0-.7-.3-1l-1-.2h-1.7-.2v-.3-.2H257c.5 0 1 0 1.2-.4.4-.2.5-.5.5-1v-.6l-.6-.5c.4 0 .7.2 1 .4 0 .2.3.4.5.6 0 .3.3.5.4.8v1zm-2.5-1.7v.5c0 .2 0 .3-.2.3h-.2v.2h-.3l-.2-.2v-.3-.5-.5-.3h.7v.3c.2.2.2.3.2.5z"/></g><g><path fill="#FFF" d="M227 283.4h499.6v110H227z"/><path fill="#005689" d="M227 283.4h112v110H227z"/><path fill="none" d="M361.5 301.4h217v15.8h-217z"/><text transform="translate(361.519 315.618)" fill="#333" font-family="'ArialMT'" font-size="19.9">The Guardian</text><path fill="none" d="M361.5 326.7h332.8V358H361.5z"/><text transform="translate(361.519 338.088)" fill="#333" font-family="'ArialMT'" font-size="15.9">Obama plan for trade deals with Asia and</text><text transform="translate(361.519 357.185)" fill="#333" font-family="'ArialMT'" font-size="15.9">Europe in tatters after Senate vote</text><path fill="none" d="M361.5 365.7h130.3v12.5H361.5z"/><text transform="translate(361.519 377.127)" fill="#BDBDBD" font-family="'ArialMT'" font-size="15.9">theguardian.com</text><path fill="#767676" d="M713 305.7l-8.5-8.4 1.5-1.5 8.4 8.4z"/><path fill="#767676" d="M714.4 297.3l-8.4 8.4-1.5-1.5 8.4-8.4z"/><path fill="#FFF" d="M269.2 307.7c-2 1.5-3.6 3.3-4.7 5.4-1 2.2-1.6 4.6-1.6 7.3 0 3.2.6 6 2 8.3 1.5 2.2 3.4 4 5.8 5.4l.3.2c-.7.4-1.5 1-2.4 1.8-1 .8-1.8 1.7-2.7 2.6-1 1-1.8 2-2.4 3.2-.6 1-1 2.3-1 3.6 0 2 .8 3.7 2 5.4 1.2 1.6 3 2.7 5.8 3.3-1.4.3-2.7.7-4 1.2-1.5.4-2.7 1-3.8 1.7-1 .7-2 1.6-2.7 2.7-.8 1-1 2.4-1 4v.5c-3.2-3.2-5.7-7-7.5-11-1.8-4.2-2.7-8.7-2.7-13.5 0-3.6.5-7 1.5-10.2 1-3.3 2.4-6.3 4.2-9 2-2.8 4-5.2 6.5-7.4 2.4-2 5-4 8.2-5.3zm21.3 49.2c2 0 3.7.4 5 1.4 1 1 1.7 2.4 1.7 4 0 2-1 3.5-2.7 4.8-1.8 1.3-5.3 2-10.5 2-4.5 0-7.7-.7-9.4-2-1.8-1.4-2.7-3-2.7-4.6 0-1.3.3-2.6 1-3.8.8-1.2 2.3-2 4.5-2h13zm28.3-17.3c0 5-1 9.5-3 13.7-1.8 4.3-4.4 8-7.6 11.2 1.3-2.4 2-5.2 2-8.4 0-4.7-1.4-8.2-4-10.4-2.6-2.2-6.5-3.3-12-3.3h-17c-1 0-1.8-.3-2.5-1-.7-.6-1-1.3-1-2 0-.8.2-1.4.6-2 .4-.7 1-1.2 1.5-1.7 1.3.5 2.6.8 3.8.8l4.2.2c6.2 0 11.2-1.5 15-4.3 3.7-3 5.6-7 5.6-12.3 0-3-.7-5.5-2-7.6-1-2.2-2.7-4-4.8-5.2 3 1.4 6 3.2 8.5 5.3 2.7 2.2 5 4.6 6.8 7.4 2 3 3.3 6 4.4 9.2 1 3.3 1.6 6.8 1.6 10.5zM290 320.3c0 2.4 0 4.3-.4 5.7-.4 1.4-.8 2.5-1.4 3.2-.6.8-1.3 1.2-2 1.4l-2.5.3c-.8 0-1.6-.2-2.4-.4-.8-.2-1.5-.6-2-1.4-.7-.7-1.2-1.7-1.5-3-.3-1.5-.4-3.4-.4-6 0-2.3 0-4.3.4-5.8.3-1.4.8-2.6 1.4-3.3.6-.7 1.3-1.2 2-1.4 1-.3 1.7-.4 2.5-.4 1 0 1.8.2 2.6.4.7.3 1.4.8 2 1.6.5.8 1 2 1.3 3.4.3 1.4.5 3.4.5 5.7z"/></g></svg>
```

Comment:
you need to remove the `xmlns="http://www.w3.org/2000/svg"` attribute to stop IE8 going insane